vividus-plugin-applitools/src/main/java/org/vividus/visual/eyes/service/ImageVisualTestingService.java

```diff
@@ -46,6 +46,8 @@ public class ImageVisualTestingService implements VisualTestingService
     private final ScreenshotProvider screenshotProvider;
     private final IHttpClient httpClient;
 
+    private boolean apiUnauthorized;
```

Comment:
it shouldn't be a field of singleton, since we can specify keys in scenarios, please, use a local variable